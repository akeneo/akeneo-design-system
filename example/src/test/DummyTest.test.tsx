import { render } from "@testing-library/react"
import "@testing-library/jest-dom";
import DummyTest from "./DummyTest.tsx"

test('renders without error', () => {
  render(<DummyTest />);
});

test('renders the expected elements', () => {
  const { baseElement } = render(<DummyTest />);
  expect(baseElement).toMatchSnapshot();
});