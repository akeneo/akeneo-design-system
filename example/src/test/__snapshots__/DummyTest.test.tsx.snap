// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the expected elements 1`] = `
<body>
  <div>
    <span
      class="sc-ghWlax dWQSyv"
    >
      Hello world
    </span>
  </div>
</body>
`;
